extensions/amp-animation/0.1/web-animations.js

```diff
@@ -879,6 +891,11 @@ class CssContextImpl {
     this.viewportSize_ = null;
   }
 
+  /** @return {!Document|!ShadowRoot} */
+  get rootNode() {
```

Comment:
This was accessed as the private method from an external user. Removing internal references would cause lint to think the member was unused, so we let it be accessed publicly.